### PR TITLE
REL-3469: Updating P1-monitor build.sbt file.

### PR DESCRIPTION
**Pull request by sraaphorst**
I always forget to do this with the other steps...

### CODE REVIEW

**Comment by cquiroz** [comment]
Do we have the `conf-production.2019.xml` file?

**Comment by sraaphorst** [comment]
Yes... I did that one separately.
It always takes me 2-3 PRs to get it all done right :-). Now I have a detailed list in Evernote, so hopefully next time I'll get all the file changes done on the first try.

File: app/p1-monitor/build.sbt [hunk 1]
```diff
@@ -113,3 +88,15 @@ def v2018B(ov: Version, pv: Version) = AppConfig(
...
   distribution = List(Linux32, Linux64)
 ) extending List(common(ov, pv))
+
+// 2019A
+def v2019A(ov: Version, pv: Version) = AppConfig(
+  id = "2019A",
+  props = Map(
+    "org.osgi.framework.storage" -> "${user.home}/.edu.gemini.p1monitor.2019A/felix-cache",
+    "p1monitor.config" -> "conf.production-2019A.xml",
+    "org.osgi.service.http.port" -> "9006",
```
Comment (cquiroz) [inline app/p1-monitor/build.sbt]: Remember to proxy the 9006 port on apache

**Comment by sraaphorst** [comment]
Thanks 😄 . I already have Apache configured for 9006 in the conf file and all should be good to go!

**Pull request merged**